### ISSUE TITLE
this._expiredKeysInterval prevents node process from closing

### ISSUE THREAD

**Issue by leviwheatcroft** [simonlast/node-persist#76]
If you call node-persist with the default options the handle for this._expiredKeysInterval prevents the node process from exiting normally.

You can call `nodePersist.initSync({expiredInterval: false})` to avoid this but I don't think it's in the docs.

**Comment by johnofleek** [comment]
I tried this workaround and it didn't work for me.

`
var storage = require('node-persist');

storage.initSync();

storage.setItemSync('key', "value");

storage.initSync({expiredInterval: false});  // commenting this out same result - node doesn't exit
`

node doesn't exit - any ideas?

**Comment by akhoury** [comment]
what do you mean by `node doesn't exit`?

**Comment by johnofleek** [comment]
akhoury I updated my note above

**Comment by leviwheatcroft** [comment]
@akhoury `node doesn't exit` means that when you run `node test.js` or so, you're never returned to the command line, the node process remains open. It's because this module calls setInterval which creates a handle every interval.

@johnofleek your first call to `storage.initSync()` would create the _expiredKeysInterval handle, which is not destroyed by your second call to `storage.initSync`. The workaround I described would look like this:

```
var storage = require('node-persist');
storage.initSync({expiredInterval: false});
storage.setItemSync('key', "value");
```

**Comment by akhoury** [comment]
@leviwheatcroft right and thanks. @johnofleek you dont need to call `initSync` twice, @leviwheatcroft's example is correct

**Comment by akhoury** [comment]
published `node-persist@2.0.8` with @noahtkeller's fix
